src/NServiceBus.Metrics.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -25,9 +25,8 @@ namespace NServiceBus
     {
         public static NServiceBus.MetricsOptions EnableMetrics(this NServiceBus.EndpointConfiguration endpointConfiguration) { }
     }
-    public class MetricsOptions
+    public class MetricsOptions : NServiceBus.Configuration.AdvancedExtensibility.ExposeSettings
     {
-        public MetricsOptions() { }
```

Comment:
I'm having a hard time thinking that we need to deprecate the implicit removal of the parameterless constructor given it should only have been instantiated by settings.GetOrCreate<T> before.